### DISCUSSION TITLE
页面比例在打开输入法之后输入框位置发生变化

### DISCUSSION THREAD

**Discussion by BrtMeister** [ly525/luban-h5#226] [Q&A]
![Screenshot_2020-09-06-14-21-29-013_com tencent mm](https://user-images.githubusercontent.com/28035340/92319581-7c312e00-f04c-11ea-8d84-70c9603baad1.jpg)

**Comment by ly525** [comment]
@BrtMeister 建议将这个转到 issue 吧
